boofuzz/blocks/size.py

```diff
@@ -220,7 +220,10 @@ def __repr__(self):
         return "<%s %s>" % (self.__class__.__name__, self._name)
 
     def __len__(self):
-        return self.length
+        if self.format == "binary":
```

Comment:
I feel like returning `self.length` in the "default" else case might be better, as the binary format is the blocks default too.

Also, when someone defines a none existing format, we would return `self.length`. Not that we'd have to cover this edge case, but I feel like this is more straight forward than the ascii format.

However this is all about personal preference I guess, so I'm fine either way.

Reply:
Yes, this makes sense. Although I think to keep it consistent, this same change should be made to bit_field. What do you think?

Reply:
Good point, we should indeed change it at both points consistently.